### DISCUSSION TITLE
dataset about dual task

### DISCUSSION THREAD

**Discussion by qoqoh** [Project-MONAI/MONAI#7276] [Q&A]
Dear all, 

my dataset should be like 
```
"training": [
    {
        "image": "imagesTr/img0001.nii.gz",
        "label": "labelsTr/label0001.nii.gz", 
        "class": 1
    },
    {
        "image": "imagesTr/img0002.nii.gz",
        "label": "labelsTr/label0002.nii.gz", 
        "class": 0
    },
    {
        "image": "imagesTr/img0003.nii.gz",
        "label": "labelsTr/label0003.nii.gz", 
        "class": 1
    },
```

Is it possible to apply the function like `LoadImaged(keys=["image", "label"])` or other functions in monai to make a (custom) dataset? If it can only be implemented by using `monai.data.Dataset`, how to apply the transform to "image" and "label" images? If so, could anyone show an example for me? 
Thanks a lot in advance!

**Comment by KumoLiu** [comment]
Hi @qoqoh, you can find more examples in the tutorial repo.
https://github.com/Project-MONAI/tutorials/tree/main/3d_segmentation

Thanks!

**Reply by qoqoh** [reply]
Hi @KumoLiu, thank you for your reply! I did check the repo you mentioned. They are all functions for reading image data (image and segmentation label) . But for me, I want to make it read the image data with its class (image, segmentation label and its class ).

**Reply by KumoLiu** [reply]
Then you could refer to the classification tutorial. There is no big difference. 
https://github.com/Project-MONAI/tutorials/blob/main/3d_classification
Thanks!

**Reply by qoqoh** [reply]
OK, thank you!
